inc/views/base/usercp/subscribe_thread.twig

```diff
@@ -0,0 +1,57 @@
+{% extends 'layouts/master.twig' %}
+
+{% block head %}
+    <title>{{ mybb.settings.bbname }} - {{ lang.forum_subscriptions }}</title>
+{% endblock head %}
+
+{% block body %}
+    <form action="usercp2.php" method="post" name="input">
+        <input type="hidden" name="my_post_key" value="{{ mybb.post_code }}" />
```

Comment:
I wonder if we should move this into a partial template since it's so commonly used...

Reply:
I agree. However, I feel like it should be done when all templates are converted.